command/ca/revoke.go

```diff
@@ -202,6 +202,10 @@ func revokeCertificateAction(ctx *cli.Context) error {
 	// If cert and key are passed then infer the serial number and certificate
 	// that should be revoked.
 	if len(certFile) > 0 || len(keyFile) > 0 {
+		// Must be using cert/key flags for mTLS revoke so should be 0 cmd line args.
+		if ctx.NArg() > 0 {
+			return errors.Errorf("'step ca revoke --cert <certificate> --key <key>' expects no additional positional arguments")
```

Comment:
you should use `ctx.App.Name` and `ctx.Command.Name` instead of `step ca revoke`

Reply:
werd. fixed